app/views/layouts/base.html.erb

```diff
@@ -57,7 +57,7 @@
       <% end %>
       <div
         id="rails-app-content"
-        class="container-fluid container-pf-nav-pf-vertical nav-pf-persistent-secondary"
```

Comment:
This is another cause of issues, as we target the container-fluid in a few places. 
If this is something that you can't live without we can open a separate PR to amend that in Katello.